deps/parselibs.jl

```diff
@@ -0,0 +1,42 @@
+function execute(cmd::Cmd)
+    out = Pipe()
+    err = Pipe()
+  
+    process = run(pipeline(ignorestatus(cmd), stdout=out, stderr=err))
+    close(out.in)
+    close(err.in)
+  
+    (
+      stdout = String(read(out)), 
+      stderr = String(read(err)),  
+      code = process.exitcode
+    )
+end
+
+parsepath = abspath(joinpath(@__DIR__,"..","etc","parse_libs.sh"))
```

Comment:
`parse_libs.sh` seems to be only there to invoke `<PATH>/Singular/libparse`, and then pass its output through awk and tail. That seems needlessly complicated. Why not call `libparse` directly, and do the parsing in Julia? 

On the other hand, `parselibs.jl` doesn't seem to be used (yet?)...

Reply:
It is WIP. I will set the paths correctly. Unfortunately, I had to rip apart my `build.jl`, so I did not commit changes in it.

Reply:
Well, it was somehow more convenient for me to make the initial parsing with `awk` and not with Julia, as I have no idea how to handle `CSV`-style stuff with Julia. I could probably do without the shell script, though.